docs/templates/tutorial-template.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Start with a verb ending in -ing
+content_type: tutorial
+
+# Optional values. Uncomment any that apply to your document.
+
+# no_version: true # Disables the version selector dropdown. Set this on pages that belong to doc sets without versions like /konnect/.
+# alpha: true # Labels the page as alpha; adds a banner to the top of the page.
+# beta: true # Labels the page as beta; adds a banner to the top of the page.
+---
+
+Add an introduction paragraph here. Good introductions explain who this tutorial is for and what this tutorial will help the user accomplish and learn.
+
+Keep the conceptual information to a minimum; the user should be learning by doing.
+
+<!-- See https://documentation.divio.com/tutorials/ for more info about how to write a tutorial -->
+<!-- See the following examples of tutorial documentation:
+* https://docs.konghq.com/gateway/latest/get-started/services-and-routes/
+* https://docs.konghq.com/gateway/latest/migrate-ce-to-ke/
+* https://docs.konghq.com/gateway/latest/kong-enterprise/analytics/influx-strategy/ 
+-->
+<!-- Remove these comments once you are done writing -->
+
+## Prerequisites <!-- Optional -->
```

Comment:
@Guaris I like that idea of having prereqs installed as part of the tutorial, but I don't know if all our current tutorial docs follow that rule. I'm okay with making that the rule going forward though. Maybe something to discuss during style + standards meeting? Or do you think the fact that Divio doesn't use them is enough to say that this should be the rule?

Reply:
I think it is okay to make that a rule going forward. How-to should have the opposite ruling, and things that need to be installed should be included in prereqs and not be installed in the how-to unless they directly relate to the topic.

Reply:
Most of our docs don't follow the rules in all of these templates at the moment :) it's definitely a going-forward initiative. 

Agree on @Guaris 's assessment overall. I think it breaks down like this:
* how-to topic: steps are out of context, and therefore require a number of prerequisites to accomplish
* tutorial: steps are telling a story/following a workflow. if there's a prereq, it's usually "follow the previous guide in this tutorial", otherwise most of the setup happens in the tutorial itself. 
  * Exceptions: external tools needed to complete a tutorial. We're not going to teach them how to install jq, for example.